File: velog_backend/src/main/java/com/velog/velog_backend/controller/TestController.java
package com.velog.velog_backend.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class TestController {

    @GetMapping("ping")
    public String pong() {
        return "pong";
    }

}

File: velog_admin/src/main/java/com/velog/controller/TestController.kt
package com.velog.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class TestController {

    @GetMapping("/")
    fun test(): ApiResponse<String> {
        return ApiResponse.OK
    }

    @GetMapping("/ping")
    fun pong(): ApiResponse<String> {
        return ApiResponse.success("pong");
    }


}